gym-kinova-gripper/gym_kinova_gripper/envs/kinova_gripper_env.py

```diff
@@ -33,112 +32,150 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import xml.etree.ElementTree as ET
-from classifier_network import LinearNetwork, ReducedLinearNetwork
 import re
 from scipy.stats import triang
-import copy # Used to copy coordinate values from the environment
+import copy  # Used to copy coordinate values from the environment
 import csv
 import pandas as pd
 from pathlib import Path
-import threading #oh boy this might get messy
-from PIL import Image, ImageFont, ImageDraw # Used to save images from rendering simulation
+import threading  # oh boy this might get messy
+from PIL import Image, ImageFont, ImageDraw  # Used to save images from rendering simulation
 import shutil
 
+import xml.etree.ElementTree as ET
+
 device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
 
+def rotate_around_origin(vec, how_much_rotation):
```

Comment:
So long as these aren't being used anywhere yet, im fine with merging

Reply:
I will remove and keep in a local file until they find use.